capa/main.py

```diff
@@ -500,7 +498,7 @@ def get_workspace(path, format_, sigpaths):
     else:
         raise ValueError("unexpected format: " + format_)
 
-    viv_utils.flirt.register_flirt_signature_analyzers(vw, sigpaths)
+    viv_utils.flirt.register_flirt_signature_analyzers(vw, [s.as_posix() for s in sigpaths])
```

Comment:
ideally we should update viv-utils to accept Path here, but thats another piece of work.

Reply:
Can create a seperate PR for this which can be merged later untill Path types are updated viv_utils package. Will look into converting paths in viv_utils.